app/views/spree/shared/_order_note.html.haml

```diff
@@ -0,0 +1,4 @@
+%p.callout{style: "margin-top: 30px"}
```

Comment:
Why inline styles? I think it's better to have these in the stylesheet itself.

Reply:
Oh I agree that its generally better to add these in the stylesheet but I was just following the pattern here. 🤣

Reply:
Yes, I think that you are both right. The special case here is that this HTML is used to generate a PDF and the styles are not re-used anywhere. So it's okay. I don't actually know if separate CSS works but it would be nice.

Reply:
Ah I see. It's okay then I think :smile: